WordPress/src/main/java/org/wordpress/android/ui/main/MySiteFragment.kt

```diff
@@ -280,7 +280,10 @@ class MySiteFragment : Fragment(),
         if (scanScreenFeatureConfig.isEnabled() || backupScreenFeatureConfig.isEnabled()) {
             uiScope.launch {
                 val itemsVisibility = jetpackCapabilitiesUseCase.getJetpackPurchasedProducts(site.siteId)
-                row_scan.setVisible(scanScreenFeatureConfig.isEnabled() && itemsVisibility.scan)
+                row_scan.setVisible(
```

Comment:
1) Minor (🔍) It seems that this class can be reformatted. 
2) Suggestion (💡): This `&&` logic is duplicated here and within the `ScanAndBackupSource` class. I suggest extracting this duplicated logic as it might cause problems in the future where a developer will forget updating one or the other. Wdyt?

PS: I would have extracted the logic into utility functions within the `ScanAndBackupSource` class and reuse those functions here. But, I am sure there are better ways, this was just the first thing that came to my mind.

Reply:
> It seems that this class can be reformatted.

This class is going to be removed soon - when the ImprovemedMySiteFragment is finished. 

> This && logic is duplicated here and within the ScanAndBackupSource class. I suggest extracting this duplicated logic as it might cause problems in the future where a developer will forget updating one or the other. Wdyt?

Definitely a good comment! I thought about it and I'd agree in general, but since this class is going to be removed in 2-4 weeks I decided to not worry about the copy paste. The only shared place is the JetpackCapabilitiesUseCase and this logic doesn't fit there. Introducing a new class which we could share felt like an overkill. Wdyt?

Reply:
Oh, you are right, maybe it is not worth it after all, thanks for considering! 🤗

Reply:
Actually, thinking about it -> I could move it to SiteUtils. Wdyt?

Reply:
I would personally do it, just to make sure that I don't get in trouble in between, even if that is for 2-4 weeks. But, it is up to you.😄

Reply:
Done in https://github.com/wordpress-mobile/WordPress-Android/pull/13962/commits/4376c3ed5c1e762cd92aa9cbe400509ba8446333